src/storage.c

```diff
@@ -0,0 +1,15 @@
+#include "storage.h"
+
+uint32_t get_secret() {
+    if (N_storage.secret == 0) {
```

Comment:
Does Ledger ensure it will be initialized with zero bytes? Or should we set it somewhere else and it will be written during app install?

Reply:
The documentation does claim that global non-const variables are zero initialized, only that the initializers on declaration are ignored and the initialization should be done on code.

When I tested and got a zero value every time (3 reinstall cycles). I imagine the storage is zeroed when the app is installed but even if it isn't, it would be initialized with trash which can be used as a random number without problems.

Reply:
Can we set an initial value to be written during app installation? It seems safer.

Even though a random value might be there, I wouldn't rely on them. I feel it opens room for some attack vectors.

Reply:
Done, made sure the installation will always be zero initialized.
The value can be anything and the initializer is ignored except on instalation, tested this and it is consistent.